lib/src/builders/filterBuilder.ts

```diff
@@ -644,6 +653,150 @@ export class FilterBuilder {
         return this;
     }
 
+    /**
+    * Adds a product variant filter to the request
+    * @param products 
+    * @param negated 
+    */
+    public addProductAndVariantIdFilter(products: ProductAndVariantId | ProductAndVariantId[], negated: boolean = false): this {
+        const filter: ProductAndVariantIdFilter = {
+            $type: 'Relewise.Client.Requests.Filters.ProductAndVariantIdFilter, Relewise.Client',
+            productAndVariantIds: Array.isArray(products) ? products : [products],
+            negated: negated,
+        };
+        this.filters.push(filter);
+
+        return this;
+    }
+
+    /**
+    * Adds a product category level filter to the request
+    * @param levels 
+    * @param negated 
+    */
+    public addProductCategoryLevelFilter(levels: number | number[], negated: boolean = false): this {
+        const filter: ProductCategoryLevelFilter = {
+            $type: 'Relewise.Client.Requests.Filters.ProductCategoryLevelFilter, Relewise.Client',
+            levels: Array.isArray(levels) ? levels : [levels],
+            negated: negated,
+        };
+        this.filters.push(filter);
+    
+        return this;
+    }
+
+    /**
+    * Adds a product category has parent filter to the request
+    * @param categoryIds 
+    * @param negated 
+    */
+    public addProductCategoryHasParentFilter(categoryIds?: string | string[], negated: boolean = false): this {
+        const filter: ProductCategoryHasParentFilter = {
+            $type: 'Relewise.Client.Requests.Filters.ProductCategoryHasParentFilter, Relewise.Client',
+            categoryIds: categoryIds ? (Array.isArray(categoryIds) ? categoryIds : [categoryIds]) : undefined,
```

Comment:
@RAORelewise Man kan godt lade vær med at angive et `categoryId` til alle de her nye filtrer, jeg vil bare gerne lige dobbelt checke at det faktisk giver mening at gøre. Giver det mening eller skal vi enforce noget her?

Reply:
Ja præcis, hvis man intet angiver så siger man bare "Denne kategori skal bare have mindst én forældre"